logos_product_attributes/models/product_attribute_value.py

```diff
@@ -0,0 +1,18 @@
+from odoo import models, api
+
+
+class ProductAttributeValue(models.Model):
+    _inherit = "product.attribute.value"
+
+    @api.multi
+    def name_get(self):
```

Comment:
Here the method should be this way
`context = dict(self._context or {})
        context.update({'show_attribute': False})
        return super(
            ProductAttributeValue, self.with_context(context)).name_get()`